### PR TITLE
Update for zDLC 3.2.0

### PR DESCRIPTION
**Pull request by cjvolzka**
Signed-off-by: Charles Volzka <cjvolzka@us.ibm.com>

### CODE REVIEW

File: code/deep_learning_compiler_run_model_example.cpp [hunk 1]
```diff
@@ -92,9 +92,11 @@ OMTensorList* generate_input(std::vector<OMTensor *>& input_tensor_vector){
         input_tensor_vector.push_back(input_tensor);
     }
 
-    // Use CreateWithOwnership(false) in this case so the OMTensors are NOT
-    // destroyed with the OMTensorList object. They will be implicitly
-    // destroyed when the input_tensor_vector falls out of its original scope.
+    // Use CreateWithOwnership(false) so the input_tensor_vector.data() array of
+    // pointers is NOT destroyed with the OMTensorList object. It will be destroyed
+    // when the input_tensor_vector falls out of its original scope.
+    // Note that the The OMTensors objects which that array points to are destroyed
```
Comment (messerb5467) [inline code/deep_learning_compiler_run_model_example.cpp:98]: Please remove the 2nd "The" in:
"Note that the The"

**Review by messerb5467 — CHANGES_REQUESTED** [review]
Please fix the one comment before we submit.

**Review by messerb5467 — APPROVED** [review]
Good to go.

**Comment by weonyuan** [comment]
Looks good to me too

**Pull request merged**